README.md

```diff
@@ -1,2 +1,16 @@
 # web-scraper-helper
 A tool to test web scraping rules.
+
+## Description
+
+This tool is a Tampermonkey script. You need to install the [Tampermonkey](https://chrome.google.com/webstore/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo) extension in your Chrome browser to use it.
```

Comment:
We should describle the use case, and move the tempermonkey stuff to the dependencies section.